hs/t/y/scientific_notation.rsh

```diff
@@ -0,0 +1,11 @@
+'reach 0.1';
+
+export const main =
+  Reach.App(
+    {},
+    [Participant('A', {})],
+    (A) => {
+      A.only(() => {
+        const x = 10e+2;
```

Comment:
You should add more examples of the things you expect to allow. Maybe include the exact example for ne1. Also, when you write new Reach examples, you should use the modern non-ternary style